### ISSUE TITLE
mount more java classes

### ISSUE THREAD

**Issue by retrogradeorbit** [epiccastle/spire#63]
```
$ spire -e 'java.net.Socket'
Exception in thread "main" clojure.lang.ExceptionInfo: Could not resolve symbol: java.net.Socket [at line 1, column 1] {:type :sci/error, :line 1, :column 1}
```

Would be nice to have many more java classes present like in babashka

**Comment by jeroenvandijk** [comment]
@retrogradeorbit Do you have a list of essentials classes? I see two long term options: 1) integrate as pod, 2) extract a common layer from babashka and use that here. 

Maybe both options are premature and just copying the implementation is easier

**Comment by retrogradeorbit** [comment]
@jeroenvandijk I just took the list from babashka and listed them as a plain literal with no special generation. Podification will be undertaken at some point but it requires a fair bit of work as much of spires internals convey object instances (which are in a separate heap in a pod and can't traverse the edn boundary). It can be done by passing id's instead, but it's a fair amount of rework, so this will suffice for the time being.

**Comment by retrogradeorbit** [comment]
fixed in 0.1.0-alpha.13
